src/radical/pilot/agent/executing/popen.py

```diff
@@ -143,13 +126,18 @@ def work(self, tasks):
                 self._prof.prof('unschedule_start', uid=task['uid'])
                 self.publish(rpc.AGENT_UNSCHEDULE_PUBSUB, task)
 
+                task['control'] = 'tmgr_pending'
+                task['$all']    = True
```

Comment:
The same is set at the staging output phase, should it be there only? (we give a control back to TMGR after staging output is passed)

Reply:
This task is failed and will not go through output staging, that's why we have to set it here.